src/config.ts

```diff
@@ -47,6 +48,14 @@ const config: Config = {
   NMDC_SERVER_API_URL:
     env.VITE_NMDC_SERVER_API_URL || "https://data-dev.microbiomedata.org",
 
+  /**
+   * The base URL of the NMDC submission-schema.
+   *
+   * Default: `https://microbiomedata.github.io/submission-schema`
+   */
+  NMDC_SCHEMA_BASE_URL:
+    env.VITE_NMDC_SCHEMA_BASE_URL || "https://microbiomedata.github.io/submission-schema",
+
```

Comment:
I have a concern about the naming of some things here. 

When I think of "the schema" or "the NMDC schema", I usually think of the schema stored in:
- https://github.com/microbiomedata/nmdc-schema

When I think of "the submission schema", I usually think of the schema stored in:
- https://github.com/microbiomedata/submission-schema

They are two different schemas.

Based on the comment (thanks for including that!) and the default value, I think you wanted this to contain the base URL of the submission schema documentation.

With that in mind, I have this suggestion for a change:

```suggestion
  /**
   * The base URL of the NMDC submission-schema documentation website.
   *
   * Default: `https://microbiomedata.github.io/submission-schema`
   */
  NMDC_SUBMISSION_SCHEMA_DOCS_BASE_URL:
    env.VITE_NMDC_SUBMISSION_SCHEMA_DOCS_BASE_URL || "https://microbiomedata.github.io/submission-schema",

```
In this suggestion, I edited the statement in the comment, the config variable name, and the `VITE_` environment variable name.